spec/controllers/content_items_controller_spec.rb

```diff
@@ -48,7 +48,7 @@
       let(:organisation) { create(:organisation_with_content_items, content_items_count: 1) }
 
       before do
-        get :show, params: { organisation_slug: organisation.slug, id: organisation.content_items.first.id }
+        get :show, params: { organisation_id: organisation.content_id, id: organisation.content_items.first.id }
```

Comment:
Not really to do with your work but this test for #show doesn't need organisation at all - just needs a content item.